packages/round-manager/src/features/program/CreateProgramPage.tsx

```diff
@@ -78,74 +78,72 @@ export default function CreateProgram() {
+              {isLoading && <p className="text-orange-500">⌛ Deploying contract to Goerli + awaiting 1 confirmation...</p>}
+              {isSuccess && <p className="text-green-600">✅ Congratulations! your grant program was successfully created!</p>}
+              {(isIPFSError || isProgramError) && <p className="text-rose-600">Error: {JSON.stringify(ipfsError || programError)}!</p>}
+            </div>
+
           </div>
 
-        </div>
-
-        <div className="col-span-2">
-
-          <form
-            className="grid grid-cols-1 gap-4 sm:items-start sm:border sm:border-gray-200 py-5 sm:px-10"
-            onSubmit={handleSubmit(onSubmit)}
-          >
-
-            <div className="sm:flex sm:flex-rows">
-              <div className="sm:basis-2/3">
-                <label htmlFor="name" className="block text-xs font-medium text-gray-700">Name</label>
-                <Input
-                  {...register("name", { required: true })}
-                  $hasError={errors.name}
-                  type="text"
-                  disabled={isLoading}
-                  placeholder="Program name"
-                />
-                {errors.name && <p className="text-sm text-red-600">{errors.name?.message}</p>}
+          <div className="col-span-2">
+
+            <form
+              className="grid grid-cols-1 gap-4 sm:items-start sm:border sm:border-gray-200 py-5 sm:px-10"
+              onSubmit={handleSubmit(onSubmit)}
+            >
+
+              <div className="sm:flex sm:flex-rows">
+                <div className="sm:basis-2/3">
+                  <label htmlFor="name" className="block text-xs font-medium text-gray-700">Name</label>
+                  <Input
+                    {...register("name", { required: true })}
+                    $hasError={errors.name}
+                    type="text"
+                    disabled={isLoading}
+                    placeholder="Program name"
+                  />
+                  {errors.name && <p className="text-sm text-red-600">{errors.name?.message}</p>}
+                </div>
               </div>
-            </div>
 
 
-            <div>
-              <label htmlFor="operators" className="block text-xs font-medium text-gray-700">
-                Additional Operator Wallets
-              </label>
+              <div>
+                <label htmlFor="operators" className="block text-xs font-medium text-gray-700">
+                  Additional Operator Wallets
+                </label>
 
-              <ul>
-                {fields.map((item, index) => {
-                  return (
+                <ul>
+                  {fields.map((item, index) => (
                     <li key={item.id} className="flex flex-rows">
                       <Input
-                        {...register(`operators.0.wallet`)}
-                        // {...register(`operators.${index}.wallet`)}
+                        {...register(`operators.${index}.wallet` as "operators.0.wallet")}
```

Comment:
you can avoid this type assertion by updating the `FormData` type to
```
type FormData = {
  name: string;
  operators: { wallet: string }[];
}
```

before the `FormData` type actually only allows exactly 1 `{ wallet: string }` in the `operators` array

Reply:
i pushed this change in commit [a088c91](https://github.com/gitcoinco/grants-round/pull/214/commits/a088c9166c93afaeabfcfc784f4859ebb183f354) here